cirq/google/arg_func_langs.py

```diff
@@ -17,4 +17,5 @@
 import sympy
 
```

Comment:
I'd define `arg_func_language` from program.proto here as a constant, and possibly describe it (or move functionality that does serialization and deserialization here).